src/database/migrations/0001_create_canonical_fields.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE canonical_fields (
+  id INTEGER PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  label TEXT,
+  short_code TEXT,
+  field_type TEXT,
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW()
+);
+
+comment on table canonical_fields is
+  'Canonical fields are those fields that are unique across organizations.';
+comment on column canonical_fields.label is 'The description of the field';
+comment on column canonical_fields.short_code is 'The short name of the field';
+comment on column canonical_fields.field_type is 'The data type of the field';
```

Comment:
Should we actually just name this `data_type`?  (I used `field_type` since I think `type` was a reserved word, but I didn't like it.  data_type seems more descriptive and also removes the need for the comment).